client/src/legacy/CMSMain.js

```diff
@@ -24,20 +24,49 @@ $.entwine('ss', function ($) {
 		}
 	});
 
+  // Page edit mode
+  $('.cms-edit-form').entwine({
+      onadd: function() {
+        // Always use treeview when in page edit mode
+        localStorage.setItem('pages-view-type', 'treeview');
```

Comment:
perhaps it'd be worth using a `ss` prefix e.g. `ss.pages-view-type`
In retrospect, I should've done this for the campaign localStorage work I did earlier as well.